mappings/net/minecraft/world/biome/Biome.mapping

```diff
@@ -58,7 +58,7 @@ CLASS net/minecraft/class_1959 net/minecraft/world/biome/Biome
 		ARG 1 step
 		ARG 2 chunkGenerator
 		ARG 3 world
-		ARG 4 seed
+		ARG 4 populationSeed
```

Comment:
This seems like an unnecessary change to me.

Reply:
I think it was to distinguish it from the world seed.

Reply:
Yes, I believe it's important to distinguish.